modules/tapadIdSystem.js

```diff
@@ -0,0 +1,78 @@
+import { uspDataHandler } from '../src/adapterManager.js';
+import { submodule } from '../src/hook.js';
+import { getStorageManager } from '../src/storageManager.js';
+import * as ajax from '../src/ajax.js'
+import * as utils from '../src/utils.js';
+
+export const storage = getStorageManager();
+const cookiesMaxAge = 7 * 24 * 60 * 60 * 1000; // 60 days
+export const defaultExpirationString = new Date(utils.timestamp() + cookiesMaxAge).toString();
+export const pastDateString = new Date(0).toString();
+export const tapadCookieKey = 'tapad_id';
+export const graphUrl = 'https://realtime-graph-access-zxvhwknfeq-uc.a.run.app/v1/graph';
+
+function saveOnAllStorages(key, value, expiration = defaultExpirationString) {
+  storage.setCookie(key, typeof value === 'object' ? JSON.stringify(value) : value, expiration);
+  storage.setDataInLocalStorage(key, JSON.stringify(value));
```

Comment:
why are you checking that `value` is an object before `JSON.stringify` for cookies but not local storage?